src/issuer/SsiCredentialIssuer.Service/BusinessLogic/IIssuerBusinessLogic.cs

```diff
@@ -26,7 +26,7 @@ namespace Org.Eclipse.TractusX.SsiCredentialIssuer.Service.BusinessLogic;
 
 public interface IIssuerBusinessLogic
 {
-    IAsyncEnumerable<UseCaseParticipationData> GetUseCaseParticipationAsync();
+    IAsyncEnumerable<UseCaseParticipationData> GetUseCaseParticipationAsync(string? status);
```

Comment:
You can directly user StatusType?

```suggestion
    IAsyncEnumerable<UseCaseParticipationData> GetUseCaseParticipationAsync(StatusType? status);
```

Reply:
Here the StatusType was not used because the error messages we have to throw to the user. They are customized and with the use of the object it self it would not be possible.